tests/CMakeLists.txt

```diff
@@ -4,3 +4,18 @@ TARGET_INCLUDE_DIRECTORIES(test_edge_sensor PRIVATE ${EDGE_REQUIRE_PKGS_INCLUDE_
 TARGET_LINK_LIBRARIES(test_edge_sensor -L${PROJECT_BINARY_DIR}/src edge-sensor)
 
 INSTALL (TARGETS test_edge_sensor DESTINATION ${BIN_INSTALL_DIR})
+
+# GoogleTest requires at least C++11
+SET(CMAKE_CXX_STANDARD 11)
+
+INCLUDE(FetchContent)
+FetchContent_Declare(
+  googletest
+  URL https://github.com/google/googletest/archive/609281088cfefc76f9d0ce82e1ff6c30cc3591e5.zip
+)
+
+FetchContent_MakeAvailable(googletest)
```

Comment:
Since gbs does not support the network, so it would be blocked for a while and failed.
Please remove this line.